src/main/java/com/laytonsmith/core/events/drivers/PlayerEvents.java

```diff
@@ -860,18 +860,28 @@ public Driver driver() {
         public boolean matches(Map<String, Construct> prefilter, BindableEvent e) throws PrefilterNonMatchException {
             if(e instanceof MCPlayerInteractEvent){
                 MCPlayerInteractEvent pie = (MCPlayerInteractEvent)e;
-                MCEquipmentSlot h = pie.getHand();
                 CString hand;
+				MCEquipmentSlot h;
+				
+				try {
+					h = pie.getHand();
+				} catch(UnsupportedOperationException ex) {
+					h = null; // We are likely on a pre-1.9 server.
+				}
                 
                 if(((MCPlayerInteractEvent)e).getAction().equals(MCAction.PHYSICAL)){
                     return false;
                 }
                 
-                if(h == MCEquipmentSlot.WEAPON) {
-                   hand = new CString("main_hand", Target.UNKNOWN);
-                } else {
-                    hand = new CString("off_hand", Target.UNKNOWN);
-                }
+				if(h == null) {
```

Comment:
A "typo" usually refers to typing one word when you meant another, in this case checking if something is not null ONLY after a previous check says it is null.

I am aware that most older files are mostly spaces instead of tabs :P